src/diff/mount.js

```diff
@@ -126,6 +123,8 @@ function mountDOMElement(dom, internal, globalContext, isSvg, commitQueue) {
 		} else if (dom.data !== newProps) {
 			dom.data = newProps;
 		}
+
+		internal._dom = dom;
```

Comment:
When a Suspense component initially renders, it should behave like a Fragment and not move its children. To do this, it needs to pass its current parentDOM to the `createPortal` function. To get its current parentDOM, we need to ensure that its parent Internals have their `_dom` pointer set before recursing down the tree. These `internal._dom = dom;` lines do that.